src/Controller/Api/AdherentsController.php

```diff
@@ -39,6 +39,29 @@ public function usersAction(AdherentRepository $adherentRepository): Response
         return new JsonResponse($count);
     }
 
+    /**
+     * @Route("/count_for_referent_managed_area", name="app_adherents_count_for_referent_managed_area")
+     * @Method("GET")
+     */
+    public function adherentsCountForReferentManagedAreaAction(AdherentRepository $adherentRepository): Response
+    {
+        $count = $adherentRepository->countByGenderManagedBy($this->getUser());
+
+        array_walk($count, function (&$item) {
```

Comment:
The repository could simply answer with a value object.
It would isolate the casting/sum logic into it.

i.e : 

```
$count = new GenderCount($male, $female);
$count->getTotal();
$count->getMaleCount();
$count->getFemaleCount();
$count->getMalePercentage();
$count->getFemalePercentage();
```

Regarding the percentage, it would be more flexible to let the front handle this calculation. They could change the rounding or use raw data without asking us.

Reply:
This is done in a generic way, if one day we add another gender we will have no need to modify code.
I agree with front calculation of percentage.

Reply:
Generic but cumbersome.
At least do a private method to avoid copy/pasting.